### ISSUE TITLE
ee/countrywide addresses

### ISSUE THREAD

**Issue by ilissablech** [openaddresses/openaddresses#6002]
Estonia coords seem to over water again... Previously reported in [this issue](https://github.com/openaddresses/openaddresses/issues/5908). But with the 2022-01-28 deployment we are now seeing this issue again. 

![image](https://user-images.githubusercontent.com/25749483/153262274-8e158c82-92eb-4ed5-8e1b-bb4e0a63ca5d.png)

cc @ingalls

**Comment by ingalls** [comment]
@ilissablech Sorry about this - when you initially reported it I simply flipped the coordinates x,y=>y,x for a quick fix. Once it became apparent that this was a more widespread issue I pushed a fix for the underlying bug last month.

I then went back through all of the sources we had flipping problems with and ensured they were now correct, however `ee` somehow missed being on this list.

https://github.com/openaddresses/openaddresses/pull/6003 restores the original source coordinate order now that the fix from last month has been deployed
